.github/workflows/build-images.yml

```diff
@@ -65,8 +106,24 @@ jobs:
         echo "Status:    ${{ steps.buildx.outputs.status }}"
         echo "Flags:     ${{ steps.buildx.outputs.flags }}"
         echo "Platforms: ${{ steps.buildx.outputs.platforms }}"
-    - name: Build ${{ matrix.name }}
-      run: mvn install --projects ${{ matrix.modules }} --also-make -DskipTests ${{ matrix.args }}
+
+    - name: Login to GitHub Container Registry
```

Comment:
There is a workflow for push.

With this, it would push PR and main builds. We want push to be explicit.